### ISSUE TITLE
TN.Mem.MBMA.G.LI.90.18 broken photo link

### ISSUE THREAD

**Issue by jbodel** [Brown-University-Library/usep-data#173]

**Comment by sdigiulio** [comment]
No image exists for this inscription in the repository; was someone responsible for preparing and uploading it recently? If not, in the meantime I can put a "no image"-placeholder.

**Comment by sdigiulio** [comment]
Was there any update as to whether or not we have an image for this inscription before I put the placeholder entry?

**Comment by jbodel** [comment]
I have only a pdf with image from the museum but not a processed version.

**Comment by sdigiulio** [comment]
Revisiting this: do we have a processed image? If not, I can assign the task to one of our team members.

**Comment by jbodel** [comment]
No, the museum website has two images we should grab, one of the entire stele, one a detail of the inscription itself, which includes the age after the name.
